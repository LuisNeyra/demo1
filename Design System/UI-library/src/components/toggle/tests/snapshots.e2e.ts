import { snapIt } from '../../../test';

const component = 'toggle';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('renders correctly with no props', '<blaze-toggle>default</blaze-toggle>');
    snap('colours based on type', '<blaze-toggle type="success">default</blaze-toggle>');
    snap('should be toggle on when prop is present', '<blaze-toggle toggled type="success">default</blaze-toggle>');
  });
});
