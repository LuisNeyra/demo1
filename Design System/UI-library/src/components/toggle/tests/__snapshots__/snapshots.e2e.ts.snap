// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toggle renders colours based on type 1`] = `"<blaze-toggle type=\\"success\\" class=\\"hydrated\\"><!----><label class=\\"c-toggle c-toggle--success\\"><input type=\\"checkbox\\" aria-checked=\\"false\\"><div class=\\"c-toggle__track\\"><div class=\\"c-toggle__handle\\"></div></div>default</label></blaze-toggle>"`;

exports[`toggle renders renders correctly with no props 1`] = `"<blaze-toggle class=\\"hydrated\\"><!----><label class=\\"c-toggle\\"><input type=\\"checkbox\\" aria-checked=\\"false\\"><div class=\\"c-toggle__track\\"><div class=\\"c-toggle__handle\\"></div></div>default</label></blaze-toggle>"`;

exports[`toggle renders should be toggle on when prop is present 1`] = `"<blaze-toggle toggled=\\"\\" type=\\"success\\" class=\\"hydrated\\"><!----><label class=\\"c-toggle c-toggle--success\\"><input type=\\"checkbox\\" aria-checked=\\"true\\"><div class=\\"c-toggle__track\\"><div class=\\"c-toggle__handle\\"></div></div>default</label></blaze-toggle>"`;
